### ISSUE TITLE
[Request] Multiple outputs from chopping recipe

### ISSUE THREAD

**Issue by Sagragoth** [gigaherz/Survivalist#90]
> If you post a feature request in my issue tracker, I will take a look at it sooner or later, although I don't know if it will make it into 1.12, or I'll wait and implement during/after the port to 1.13.

Please give chopping block recipes the capability to drop multiple types of items, such as chopping a log dropping planks, sawdust, and bark, etc.

**Comment by rckymtnrfc** [comment]
Yes please!  I was just trying to tweak the .json files for chopping recipes but I couldn't figure out how to have multiple outputs from one ingredient.
